ERCS/erc-7717.md

```diff
@@ -0,0 +1,250 @@
+---
+eip: 7717
+title: Deterministic AA wallet
+description: Propose a non-custodial interface for service providers to provide AA accounts.
+author: Jinseok Park (@mellowcroc), Jaehun Kim (@jaehunkim), DrKeccak (@drkeccak)
+discussions-to: https://ethereum-magicians.org/t/eip-7710-deterministic-aa-wallet/20080
+status: Draft
+type: Standards Track
+category: ERC
+created: 2024-05-22
+requires: 4337
+---
+
+## Abstract
+
+Propose a non-custodial interface for service providers to provide AA accounts.
+
+## Motivation
+
+- Problem
+    - The `createAccount` function necessitates an `owner` address to establish an account, consequently preventing service providers from creating accounts on behalf of their users without knowledge of the users’ EOA addresses. To resolve this limitation, service providers employ temporary EOAs to deterministically generate an [ERC-4337](./eip-4337.md) account for their users, assigning the temporary EOA as the initial owner. This approach, however, introduces a security vulnerability, as the temporary EOAs’ private keys are managed by the service providers. Furthermore, the service providers assume responsibility for these private keys until such a time when users take the ownership.
+- Solution
+    - Set a separate contract as the owner when calling `createAccount`, and allow the user to claim ownership at a later date via a signature provided by the service provider.
```

Comment:
The motivation section should be written (for the most part) in normal paragraph style.